### PR TITLE
feat: add cacheControl parameter

### PR DESCRIPTION
**Pull request by norman-ags**
Solves #156

### CODE REVIEW

**Comment by coliff** [comment]
Thanks for the PR! This looks good to me, but I'd like @Reggionick to approve before merging... and then hopefully shipping a new release!

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ S3 Deploy's Action supports inputs from the user listed in the table below:
 | `private`          | boolean          | No       | false        | Upload files with private ACL, needed for S3 static website hosting
 | `cache`            | string           | No       |              | Sets the Cache-Control: max-age=X header
 | `immutable`        | boolean          | No       | false        | Sets the Cache-Control header to 'immutable'
+| `cacheControl`     | string           | No       |              | Sets the Cache-Control: X header
```
Comment (coliff) [inline README.md:35]: I think this should be `cache-control` rather than `cacheControl` (to match the name specified in `action.yml`)

Comment (Reggionick) [reply README.md:35]:
```suggestion
| `cache-control`    | string           | No       |              | Sets the Cache-Control: X header
```

**Review by Reggionick — CHANGES_REQUESTED** [review]

**Review by Reggionick — APPROVED** [review]

**Review by Reggionick — CHANGES_REQUESTED** [review]
Could you please resolve conflicts?

**Review by Reggionick — APPROVED** [review]
LGTM

**Pull request merged**